File: 2177.find-three-consecutive-integers-that-sum-to-a-given-number/solution.php
class Solution {

    /**
     * @param Integer $num
     * @return Integer[]
     */
    function sumOfThree($num) {
        if ($num % 3 != 0) {
            return [];
        }
        $num /= 3;
        return [$num-1, $num, $num+1];
    }
}

File: 2177.find-three-consecutive-integers-that-sum-to-a-given-number/solution.cpp
class Solution {
public:
    vector<long long> sumOfThree(long long num) {
        if (num % 3 != 0)
            return vector<long long>();
        num /= 3;
        long long nums[] = {num-1, num, num+1};
        return vector<long long>(nums, nums+3);
    }
};
